tests/tests_pytorch/callbacks/test_sanity_serving.py

```diff
@@ -0,0 +1,43 @@
+from typing import Dict
+
+import torch
+
+from pytorch_lightning import seed_everything, Trainer
+from pytorch_lightning.callbacks.sanity_serving import SanityServing, ServableModule
+from pytorch_lightning.demos.boring_classes import BoringModel
+
+
+class ServableBoringModel(BoringModel, ServableModule):
+    def configure_payload(self) -> ...:
+        return {"body": {"x": list(range(32))}}
+
+    def configure_inputs_outputs(self):
+        class Tensor:
+            @staticmethod
+            def deserialize(x):
+                return torch.tensor(x).float()
+
+            @staticmethod
+            def serialize(x):
+                return x.numpy().tolist()
+
+        return ({"x": Tensor.deserialize}, {"output": Tensor.serialize})
+
+    def serve_step(self, x: torch.Tensor) -> Dict[str, torch.Tensor]:
+        return {"output": self.forward(x)}
```

Comment:
would it be better to call `self(x)` instead?

Reply:
Done !